index.js

```diff
@@ -0,0 +1,5 @@
+const baseManifest = require('./base-manifest');
+
+module.exports = {
+  baseManifest
```

Comment:
could you update README.md [Quick start paragraph](https://github.com/getndazn/kopytko-packager#quick-start)?